ansible_collection/hpe/nimble/README.md

```diff
@@ -1,65 +1,46 @@
+# HPE Nimble Storage Content Collection for Ansible
 
-# HPE Nimble Storage Ansible Collection
-
-
-
-## Prerequisites
-
+This is a pre-release of the HPE Nimble Storage Content Collection for Ansible.
 
+## Requirements
 
 - Ansible 2.10 or later
-
-- HPE Nimble Storage array running NimbleOS 5.0 or later
-
+- HPE Nimble Storage arrays running NimbleOS 5.0 or later
 - HPE Nimble Storage SDK for Python
-
 - Python >=v3.6
 
-
-
-
 ## Installation
 
+Install the collection on your Ansible management host.
 
+```
+ansible-galaxy collection install git+https://github.com/hpe-storage/nimble-ansible-modules.git,rel-1.0.0
+```
 
+**Note**: This is a beta release. The content collection will be made available through Ansible Galaxy and Automation Hub upon release.
 
-Install the HPE Nimble Storage array collection on your Ansible management host.
-
-
-
-
-- Install the collection using the following command:
-`ansible-galaxy collection install git+https://github.com/hpe-storage/nimble-ansible-modules.git,rel-1.0.0`
-
-
-- Above command will install the collection in /root/.ansible/collections/ansible_collections
+## Documentation
 
+A local rendering of the Ansible 2.10 docs with the HPE Nimble Storage Content Collection for Ansible is being made available below.
 
-
-- To install in a custom folder, please use the following command:
-
-	 `ansible-galaxy collection install git+https://github.com/hpe-storage/nimble-ansible-modules.git,rel-1.0.0 -p /usr/share/ansible/collections` .
- This will install the collection under usr/share/ansible/collections.
-
-
-
-
-**Note**: This is a beta release. The above steps will finally be removed once we upload and publish our collection on galaxy server and will be replaced with the official way to install a ansible collection.
-
-
-
-## Available Modules
```

Comment:
is there a problem in having a list of available modules in README too..  ?

Reply:
Listed in updated PR.